prepsrl/src/test/java/edu/illinois/cs/cogcomp/prepsrl/PrepSRLAnnotatorTest.java

```diff
@@ -55,4 +55,15 @@ public void testLong() throws Exception {
         View view = annotator.getView(ta);
         assertEquals("Source", view.getConstituents().get(0).getLabel());
     }
+
+    @Test
+    public void testException() throws Exception {
+        String longSentence = "That put him a stroke ahead of Curt Byrum, John Adams and " +
+                "Lonnie Clements , all of whom are looking for their first " +
+                "victory on the PGA Tour .";
+        TextAnnotation ta = TextAnnotationUtilities.createFromTokenizedString(longSentence);
+        preprocessor.annotate(ta);
+        View view = annotator.getView(ta);
+        assertEquals("ahead of", view.getConstituents().get(0).getSurfaceForm());
+    }
```

Comment:
@christos-c does this pass on your machine?
In my machine it gives: 
```
https://lists.sourceforge.net/lists/listinfo/ojalgo-user
ILP ERROR: Unsatisfiable constraints!
Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 42.886 sec <<< FAILURE! - in edu.illinois.cs.cogcomp.prepsrl.PrepSRLAnnotatorTest
testException(edu.illinois.cs.cogcomp.prepsrl.PrepSRLAnnotatorTest)  Time elapsed: 0.502 sec  <<< FAILURE!
java.lang.AssertionError: ILP: Constant encountered. (false)
	at edu.illinois.cs.cogcomp.lbjava.infer.ILPInference.visit(ILPInference.java:673)
	at edu.illinois.cs.cogcomp.lbjava.infer.PropositionalConstant.runVisit(PropositionalConstant.java:260)
	at edu.illinois.cs.cogcomp.lbjava.infer.ILPInference.infer(ILPInference.java:232)
	at edu.illinois.cs.cogcomp.lbjava.infer.ILPInference.valueOf(ILPInference.java:276)
	at edu.illinois.cs.cogcomp.prepsrl.inference.ConstrainedPrepSRLClassifier.discreteValue(ConstrainedPrepSRLClassifier.java:77)
	at edu.illinois.cs.cogcomp.prepsrl.PrepSRLAnnotator.addView(PrepSRLAnnotator.java:85)
	at edu.illinois.cs.cogcomp.annotation.Annotator.lazyAddView(Annotator.java:181)
	at edu.illinois.cs.cogcomp.annotation.Annotator.getView(Annotator.java:166)
	at edu.illinois.cs.cogcomp.prepsrl.PrepSRLAnnotatorTest.testException(PrepSRLAnnotatorTest.java:66)


Results :

Failed tests: 
  PrepSRLAnnotatorTest.testException:66 ILP: Constant encountered. (false)
```
I changed the catch statement to: 
```
} catch (Exception | AssertionError e) {
```
where it's failing due to existing span. Thoughts?